dfastmi/batch.py

```diff
@@ -106,25 +111,26 @@ def batch_mode_core(
     apply_q : BoolVector
 
     display = False
+    data = RiverParameterData(config)
     
     # check outputdir
     if rootdir == "":
         rootdir = os.getcwd()
-    outputdir = dfastmi.io.config_get_str(config, "General", "OutputDir", rootdir + os.sep + "output")
+    outputdir = data.config_get_str("General", "OutputDir", rootdir + os.sep + "output")
     if os.path.exists(outputdir):
         if display:
-            dfastmi.io.log_text("overwrite_dir", dict={"dir": outputdir})
+            ApplicationSettingsHelper.log_text("overwrite_dir", dict={"dir": outputdir})
     else:
         os.makedirs(outputdir)
-    report = open(outputdir + os.sep + dfastmi.io.get_filename("report.out"), "w")
+    report = open(outputdir + os.sep + ApplicationSettingsHelper.get_filename("report.out"), "w")
 
     prog_version = dfastmi.__version__
-    dfastmi.io.log_text("header", dict={"version": prog_version}, file=report)
-    dfastmi.io.log_text("limits", file=report)
-    dfastmi.io.log_text("===", file=report)
+    ApplicationSettingsHelper.log_text("header", dict={"version": prog_version}, file=report)
+    ApplicationSettingsHelper.log_text("limits", file=report)
+    ApplicationSettingsHelper.log_text("===", file=report)
 
     cfg_version = config["General"]["Version"]
-    rvr_version = rivers["version"]
```

Comment:
I assume the rivers.version gets the correct version in comparison to the rivers["version"]

Reply:
discussed